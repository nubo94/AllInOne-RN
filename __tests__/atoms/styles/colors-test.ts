/**
 * Describe color theme
 * @format
 */

import 'react-native';
import Colors from '../../../src/atoms/styles/colors';

test('should be a object', () => {
  const expected = {
    light: {
      primary: {
        dark: '#5A6AE5',
        main: '#6C75F5',
        light: '#F3F5FF',
        ultraDark: '#475ABE',
      },
      secondary: {
        dark: '#E5D55A',
        main: '#F5EC6C',
        light: '#FFFDF3',
        ultraDark: '#BEAB47',
      },
      primaryText: {
        dark: '#212121',
        main: '#333333',
        light: '#BDBDBD',
        ultraLight: '#E0E0E0',
      },
      primaryBackground: {
        main: '#FAFAFA',
      },
      secondaryBackground: {
        main: '#FFFFFF',
      },
      divider: {
        main: '#EEEEEE',
      },
      status: {
        info: '#333',
        danger: '#f44336',
        success: '#43A047',
        warning: '#E65100',
      },
    },
    dark: {
      primary: {
        dark: '#212121',
        main: '#424242',
        light: '#424242',
        ultraDark: '#424242',
      },
      secondary: {
        dark: '#3A3509',
        main: '#888009',
        light: '#4F4200',
        ultraDark: '#453E18',
      },
      primaryText: {
        dark: '#E0E0E0',
        main: '#FFFFFF',
        light: '#BDBDBD',
        ultraLight: '#EEEEEE',
      },
      divider: {
        main: '#616161',
      },
      primaryBackground: {
        main: '#303030',
      },
      secondaryBackground: {
        main: '#424242',
      },
      status: {
        info: '#E0E0E0',
        danger: '#B71C1C',
        success: '#1B5E20',
        warning: '#FF6D00',
      },
    },
  };

  expect(Colors).toEqual(expected);
});
