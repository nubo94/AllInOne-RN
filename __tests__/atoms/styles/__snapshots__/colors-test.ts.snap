// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should be a object - colors 1`] = `
Object {
  "darker": Object {
    "divider": Object {
      "main": "#616161",
    },
    "primary": Object {
      "dark": "#212121",
      "light": "#424242",
      "main": "#424242",
      "ultraDark": "#424242",
    },
    "primaryBackground": Object {
      "main": "#303030",
    },
    "primaryText": Object {
      "dark": "#E0E0E0",
      "light": "#BDBDBD",
      "main": "#FFFFFF",
      "ultraLight": "#EEEEEE",
    },
    "secondary": Object {
      "dark": "#3A3509",
      "light": "#4F4200",
      "main": "#888009",
      "ultraDark": "#453E18",
    },
    "secondaryBackground": Object {
      "main": "#424242",
    },
    "status": Object {
      "danger": "#B71C1C",
      "info": "#E0E0E0",
      "success": "#1B5E20",
      "warning": "#FF6D00",
    },
  },
  "lighter": Object {
    "divider": Object {
      "main": "#EEEEEE",
    },
    "primary": Object {
      "dark": "#5A6AE5",
      "light": "#F3F5FF",
      "main": "#6C75F5",
      "ultraDark": "#475ABE",
    },
    "primaryBackground": Object {
      "main": "#FAFAFA",
    },
    "primaryText": Object {
      "dark": "#212121",
      "light": "#BDBDBD",
      "main": "#333333",
      "ultraLight": "#E0E0E0",
    },
    "secondary": Object {
      "dark": "#E5D55A",
      "light": "#FFFDF3",
      "main": "#F5EC6C",
      "ultraDark": "#BEAB47",
    },
    "secondaryBackground": Object {
      "main": "#FFFFFF",
    },
    "status": Object {
      "danger": "#f44336",
      "info": "#333",
      "success": "#43A047",
      "warning": "#E65100",
    },
  },
}
`;
